app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/common/PresenterBaseActivity.kt

```diff
@@ -0,0 +1,24 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
+abstract class PresenterBaseActivity : AppCompatActivity() {
```

Comment:
1. presenter is aware of lifecycle
2. every presenter do not need this
3. using inheritance to extract arbitrary logic is seriously broken
4. people will copy paste it